packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/trade_definition_market.js

```diff
@@ -58,33 +58,27 @@ Blockly.Blocks.trade_definition_market = {
         const symbol = symbol_dropdown.getValue();
 
         const populateMarketDropdown = () => {
-            active_symbols.getMarketDropdownOptions().then(market_options => {
-                market_dropdown.updateOptions(market_options, {
-                    default_value: market,
-                    should_pretend_empty: true,
-                    event_group: event.group,
-                });
+            market_dropdown.updateOptions(active_symbols.getMarketDropdownOptions(), {
+                default_value: market,
+                should_pretend_empty: true,
+                event_group: event.group,
```

Comment:
Could we refactor this snippet to a function as it's repeated 3 times?

Reply:
@salar-fs  Actually, I think the disadvantages of creating function is more here. Since I am already calling one function for 3 different fields.
If I want to create another function, I will pass an object as an input (since the input count is 5) and should pass them like I am doing now and no decrease in line number will happen. Also, I believe in that case I will decrease the readability of the code.

Reply:
Totally agree :+1: